### DISCUSSION TITLE
sending mail larger than 20 megabytes

### DISCUSSION THREAD

**Discussion by localadmin33** [postalserver/postal#3027] [Help with using Postal]
Good day, we are using postal. Works via API. When uploading to Postal, a letter larger than 10 MB is not sent. What can be done?

My postal.yml config use postal 2.2

general:  
  use_ip_pools: false
web: 
  host: mail.domain.com
  protocol: https
web_server:
  bind_address: 10.0.*.*
  port: 5000
smtp_server:
  port: 25
  max_message_size: 20
logging:
  stdout: true
main_db:
  host: 127.0.0.1
  username: root
  password: ******
  database: postal
message_db:
  host: 127.0.0.1
  username: root
  password: ******
  prefix: postal
rabbitmq:
  host: 127.0.0.1
  username: postal
  password: ******
  vhost: postalvhost
dns:
  mx_records:
    - mx1.domain.com
  smtp_server_hostname: smtp.domain.com
  spf_include: spf.domain.com
  return_path: rp.domain.com
  route_domain: routes.domain.com
  track_domain: track.domain.com
smtp:
  host: mail.domain.com
  port: 2525
  username: smtp-user
  password: *********
  from_name: Postal
  from_address: mail@domain.com

**Comment by willpower232** [comment]
What error message are you seeing? Does the message appear in the Postal UI?
